src/devices/Ad7193/Ad7193.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <Configurations>Windows-Debug;Linux-Debug;Linux-Release;Windows-Release;Debug;Release</Configurations>
```

Comment:
Project shouldn't require configurations once you have built using build.cmd once, so please go ahead and remove this line.

Reply:
@joperezr (for this and the next few comments) That was my change. And it's the only way I found that made the build pass in VS for all platforms. Without this, if you build on Windows, you get an error saying that System.Device.Gpio.dll was not found (it's using a wrong path). 
This is relevant both for running samples on Windows (for bindings where this is possible) as well as to make sure that any unit tests run on all platforms. We have many unit tests that are not executed on windows due to this (although they are technically platform-independent). I'll create separate PR/issues after further investigation, but I've just tried out this concept here as Andras has run into much the same issues as I have.

Reply:
The code for this binding is AnyCPU and is all IL that applies for all platforms, so there is no need to build it for more than one configuration/platform. If you keep it the way we have it for other bindings and make sure you first build using build.cmd/build.sh from the root, then building individual projects from VS should definitely work for all platforms, if that is not the case it is definitely a bug.

Reply:
I do think this is a bug/inconsistency in the build environment. I'll come up with a separate ticket on this, including a more extensive explanation of the problem.

Reply:
See #1037